OpenRA.Mods.Common/UtilityCommands/ReplayChatCommand.cs

```diff
@@ -0,0 +1,167 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using OpenRA.FileFormats;
+using OpenRA.Network;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	class ReplayChatCommand : IUtilityCommand
+	{
+		public string Name { get { return "--replay-chat"; } }
+		readonly int gameTicksPerSecond = 25;		// Engine
+		readonly int gameTicksPerNetworkTick = 3;	// constants
+
+		[Desc("REPLAYFILE", "Extract chat content from a replay file.")]
+		public void Run(ModData modData, string[] args)
+		{
+			if (args.Length == 1)
+			{
+				Console.WriteLine("Path argument for a replay file is missing.");
```

Comment:
The Run method is called with at least one argument. I will add checks for too many arguments.